### PR TITLE
Bump WP tested up to 6.0

### LINKED ISSUE #2583: Update "Tested up to" in our readme.txt
### Description

Test against the latest WordPress version and update the "Tested up to" WordPress version on this line:
https://github.com/Automattic/woocommerce-services/blob/trunk/readme.txt#L6
image

### PR DESCRIPTION
**Pull request by bartech**
### Related issue(s)
Closes #2583 

### Checklist

- [x] unit tests

- [ ] `changelog.txt` entry added

### CODE REVIEW

**Comment by harriswong** [comment]
Oh wow that's fast. You beat me to it. Thanks @bartech!! 🥇

File: readme.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ Contributors: woocommerce, automattic, woothemes, allendav, kellychoffman, jkudi
 Tags: shipping, stamps, usps, woocommerce, taxes, payment, dhl, labels
 Requires at least: 4.6
 Requires PHP: 5.3
-Tested up to: 5.9
+Tested up to: 6.6.1
```
Comment (harriswong) [inline readme.txt]: Isn't this 6.0.2? https://en-ca.wordpress.org/download/releases/

**Comment by harriswong** [comment]
I tested against WP 6.0.2, WC 6.9.4. 
- printed a USPS label ok
- tested refund also ok.

File: readme.txt (hunk 1 above)
Comment (bartech) [reply readme.txt]: Right, this is WP version here. Fixing.

**Review by harriswong — APPROVED** [review]
LGTM 🚢

**Pull request merged**